lib/browser.js

```diff
@@ -6,6 +6,7 @@
  *
  */
 
+import historyPolyfill from 'html5-history-api';
```

Comment:
> not sure how that would make the build smaller if were still requiring it in?

It wouldn't make the build smaller for users that choose to include the polyfill, but it would make the build smaller for users who don't have to support theses old browsers because they just won't include it. I don't like including a 10kb polyfill (minified) in my app that I don't even need. That would make Tarantino twice as big.

You're right in that Tarantino does support legacy browsers, but that is only for the hash-based router since basic support for that is already present in those browsers (albeit with some inconsistencies, which this library fixes).

Note that I'm also new to this library, so bear with me if I made any mistakes.